README.md

```diff
@@ -1 +1,10 @@
-# ntd-modernization
\ No newline at end of file
+# ntd-modernization
+  
+This repo contains analyses and code related to helping make Caltrans NTD reporting more efficient. 
+  
+### Issues Analysis, 2020 - 2022
+The `ntd_issues_analysis.Rmd` R Markdown file analyses errors Caltrans received from NTD from 2020 - 22. The accompanying HTML file shows code and web-interactive graphics (if it does not render, download onto your computer and reopen).  
+  
+---
+### Pre-submission Validator
+The folder `validator_tool` contains code for prototyping a presubmission validation tool to anticipate issues in submitting NTD reports, based on 2020-22 issues.
```

Comment:
missing `requirements.txt` or other software definition package. also missing  how to run.

Reply:
This is not a library or executable repo as-is, so the readme is more a description of what is in each folder, with some showing intermediate development work. I think as the infrastructure gets put in place, we can chunk out code sections that are executable - and in sub-areas describe how to run things?